sovtokenfees/sovtokenfees/test/chain/test_fees_chain.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+from sovtokenfees.test.helper import add_fees_request_with_address
+
+from plenum.common.exceptions import RequestRejectedException
+
+
+def send_nym_with_fees(helpers, address, change_address, fees_set, adjust_fees=0):
+    (did, verkey) = helpers.wallet.create_did()
+    request = helpers.request.nym(dest=did, verkey=verkey)
+    request = add_fees_request_with_address(helpers, fees_set, request, address, change_address=change_address, adjust_fees=adjust_fees)
+    helpers.sdk.send_and_check_request_objects([request])
+    return did
+
+
+def test_fees_chain(addresses, helpers, mint_tokens, fees_set, looper):
+    send_nym_with_fees(helpers, addresses[0], addresses[1], fees_set)
+    send_nym_with_fees(helpers, addresses[1], addresses[2], fees_set)
+    send_nym_with_fees(helpers, addresses[2], addresses[3], fees_set)
+
+    utxos = helpers.general.get_utxo_addresses(addresses[3:])
+
+    assert utxos[0][0]["amount"] == 988
```

Comment:
Why 988? Where do we set Fees? It would be great to set them explicitly in the test.